src/firebase/techCenterCheckout.Data.js

```diff
@@ -1,18 +1,19 @@
 /* eslint-disable no-unused-vars */
 import {
-  getDocs, onSnapshot, query, collection, where,
+  getDocs, onSnapshot, query, collection, where, deleteDoc, doc, getDoc,
 } from 'firebase/firestore';
 import db from './techCenterCheckout.Firestore';
 
 // Read all items from inventory
 async function getCollection() {
   const querySnapshot = await getDocs(collection(db, 'All Devices'));
   const allDevices = [];
-  querySnapshot.forEach((doc) => {
+  // used to be doc here...
```

Comment:
comment unnecessary, and I'm not sure why 'document' is favorable over 'doc'? Doc is more readable imo